x-pack/plugins/cases/server/client/utils.ts

```diff
@@ -519,6 +520,9 @@ export const convertSortField = (sortField: string | undefined): SortFieldCase =
       return SortFieldCase.title;
     case 'severity':
       return SortFieldCase.severity;
+    case 'updatedAt':
+    case 'updated_at':
+      return SortFieldCase.updatedAt;
```

Comment:
I saw `createdAt/_at` and `closedAt/_at` do the same but why do we need both conversions?

Reply:
> Then I added updatedAt and it worked 😄 And it makes sense because I added updatedAt sorting filed.

This is so weird. Most places I found use `SortFieldCase.updatedAt` which is `updated_at` so dunno where updatedAt might be passed 🤔 . Anyway, this is fine, no need to change.

Reply:
I think I found it. The rows are populated with instances of

`export type Case = Omit<SnakeToCamelCase<CaseResponse>, 'comments'> & { comments: Comment[] };`

so when adding the columns we do:
```
columns.push({
  field: 'updatedAt',
  render: (updatedAt: Case['updatedAt']) => {
```

Clicking a header of this table calls `tableOnChangeCallback` defined in `all_cases_list.tsx` row 120.

This is triggered when the sorting changes in the EuiTable([see onChange props](https://elastic.github.io/eui/#/tabular-content/tables#a-basic-table)).

So that `field: 'updatedAt'` is passed to this callback and then we need to say `sortField: getSortField(sort.field),`

https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/public/components/all_cases/all_cases_list.tsx#L126

Dunno, it looks unnecessarily complicated.

But don't change it, I am just digging around 😄